packages/features/insights/filters/OrgTeamsFilter.tsx

```diff
@@ -0,0 +1,148 @@
+import { useSession } from "next-auth/react";
+import { useState } from "react";
+
+import {
+  FilterCheckboxField,
+  FilterCheckboxFieldsContainer,
+} from "@calcom/features/filters/components/TeamsFilter";
+import { getPlaceholderAvatar } from "@calcom/lib/defaultAvatarImage";
+import { useLocale } from "@calcom/lib/hooks/useLocale";
+import { trpc } from "@calcom/trpc";
+import { AnimatedPopover, Avatar, Divider, Icon, FilterSearchField } from "@calcom/ui";
+
+export type OrgTeamsType = "org" | "team" | "yours";
+
+// This is a cleaned-up version of TeamAndSelfList.
+export const OrgTeamsFilter = ({
```

Comment:
This button is not a part of "Add Filters" button and its "ActiveFilters". It exists separately on the left-most side. 

![Screenshot 2025-01-14 at 17 35 25](https://github.com/user-attachments/assets/7a665eea-7c32-491d-a0fc-eeab468d04a0)